### PR TITLE
Cleanup check access

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/models/project.rb [hunk 1]
```diff
@@ -371,7 +371,7 @@ def self.check_access?(dbp)
         # LDAP
         # FIXME: please do not do special things here for ldap. please cover this in a generic group model.
         if CONFIG['ldap_mode'] == :on && CONFIG['ldap_group_support'] == :on
-          if UserLdapStrategy.user_in_group_ldap?(User.current, group.group_id)
+          if UserLdapStrategy.user_in_group_ldap?(User.current, grouprel.group_id)
```
Comment (bgeuken) [inline src/api/app/models/project.rb]: This makes me wonder if that part of the code got ever touched

File: src/api/app/models/package.rb [hunk 2]
```diff
@@ -115,7 +115,7 @@ class PutFileNoPermission < APIException; setup 403; end
 
   has_many :tokens, dependent: :destroy, inverse_of: :package
 
-  def self.check_access?(dbpkg = self)
+  def self.check_access?(dbpkg)
```
Comment (hennevogel) [inline src/api/app/models/package.rb]: can you also use some speaking variables if you are at it?

Comment (bgeuken) [inline src/api/app/models/package.rb]: done

**Comment by bgeuken** [comment]
@hennevogel Please review again

**Pull request merged**